src/components/NewTimeline.vue

```diff
@@ -317,7 +312,7 @@ $chevronLeft: '~@/assets/images/chevron-left.png';
 $chevronDown: '~@/assets/images/chevron-down.png';
 .custom-tabbed-accordion {
   display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
```

Comment:
Might want to stick with fractions here (fr) as that is the recommended unit for CSS grid. Other units will work but sometimes will give unexpected results, at least according to the documentation. Something like 2fr 2fr 3fr (I haven't done the exact calculations :) ) might give the extra room desired on the final tab.

Reply:
makes sense! I changed this because I wanted the bar widths to math the years they correspond to in the chart but I see how I can use that same method to achieve what I want